docs/docs/setting-up/workload-onboarding/bacalhau-docker-image/index.md

```diff
@@ -78,6 +78,32 @@ docker run -t ghcr.io/bacalhau-project/bacalhau:latest \
 
  `sh -c 'uname -a && echo "Hello from Docker Bacalhau!"'`: The command executed inside the container 
 
+### Declarative job description
+
+The same job can be presented in the [declarative](../../../setting-up/jobs/job-specification/job.md) format. In this case, the description will look like this:
+
+```yaml
+name: A Simple Docker Job
+type: batch
+count: 3
+tasks:
+  - name: My main task
+    engine:
+      type: docker
+      params:
+        Image: ubuntu:latest
+        Entrypoint:
+          - /bin/bash
+        Parameters:
+          - -c
+          - echo Hello Bacalhau!
+```
+
+The job description should be saved in `.yaml` format, e.g. `hello.yaml`, and then run with the command:
+```bash
+bacalhau job run hello.yaml
+```
```

Comment:
This page is about running Bacalhau _itself_ from within Docker. To recreate what is happening above, you need to run Bacalhau from within its own Docker image, pass this file to that image, and get it to submit the job that way. Users who are running Bacalhau within Docker won't have a bacalhau command installed locally.